### ISSUE TITLE
Improve accessibility of the Wallet Export screen

### ISSUE THREAD

**Issue by GBKS** [BlueWallet/BlueWallet#4742]
Did some quick accessibility testing on this screen and found a few issues with labelling, as follows:

- The "x" button in the top-right corner is only labelled as "Button". A user reliant on a screen reader will not know what it does. "Close" or "Done" would be better.
- The QR code is invisible to a screen reader. It should be selectable and have a label like "Scan QR code to copy recovery phrase".
- The recovery phrase is a single text field and screen readers read it out super fast like a sentence. That makes it difficult for someone to write it down. Might be better if each word is its own text field, so they can be selected one at a time.
- Alternate methods like copy or share would be helpful for users who can't see or have issues with motor skills.

Not ideal from a security perspective when a screen readers loudly announces every word of a recovery phrase, but I am not sure there is a good alternative to this. If you have ideas, would be cool to hear.

**Comment by limpbrains** [comment]
I started working on `limpbrains-ally` branch a couple of month ago, but abandoned it. Might be useful

**Comment by GBKS** [comment]
Sweet. I saw you added the `react-native-a11y/basic` module in your branch. What are the benefits of it? I couldn't find good documentation.

**Comment by limpbrains** [comment]
It is a plugin for eslint, it helps find missing labels for Buttons and fix them

**Comment by ecurrencyhodler** [comment]
@Overtorment would you mind closing this issue? Looks like it's been addressed in #5390
